pkg/autopilot/controller/readyprober.go

```diff
@@ -81,7 +81,7 @@ func (p readyProber) Probe() error {
 	g := errgroup.Group{}
 
 	for _, target := range p.targets {
-		// nolint:govet
+		target := target
 		g.Go(func() error {
 			return func(target apv1beta2.PlanCommandTargetStatus) error {
 				return p.probeOne(target)
```

Comment:
```suggestion
		currentTarget := target
		g.Go(func() error {
			return func(target apv1beta2.PlanCommandTargetStatus) error {
				return p.probeOne(currentTarget)
```

I'm fine either way but `currentTarget` may be slightly more readable.

Reply:
also, the `errorCh` in same function appears to be completely unused.

Reply:
IMO It's better to shadow the outer variable completely, as it's virtually always a bug to use it when goroutines are involved in the loop body. Go is actually considering to change the language behavior of this, because it's so error-prone: See https://github.com/golang/go/wiki/LoopvarExperiment

Reply:
> also, the `errorCh` in same function appears to be completely unused.

Good catch. Will file another PR for that.

Reply:
#3491